### PR TITLE
added focus subclass and tab index

### PR DESCRIPTION
**Pull request by Takomane**
#360 focus subclass added to testimonial-row and testimonial-text class to include a border of a thin solid black line.

### CODE REVIEW

**Comment by Takomane** [comment]
#360 added focus subclass and tab index to elements

**Comment by Takomane** [comment]
Updated tabindex and removed focus subclass for testimonial text

File: src/pages/testimonials.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ function Testimonial() {
             </div>
             <div className="col-md-12">
               <div className="cause_section_content">
-                <div className="testimonial-row">
+                <div className="testimonial-row" tabIndex="0">
```
Comment (andrew-t-james) [inline src/pages/testimonials.js:30]: Please revert tab index on non interactive elements. 

https://www.a11yproject.com/posts/how-to-use-the-tabindex-attribute/

**Review by andrew-t-james — CHANGES_REQUESTED** [review]

File: src/pages/testimonials.js (hunk 1 above)
Comment (Takomane) [reply src/pages/testimonials.js:30]: Received, I will have it done tonight!

**Review by jeromehardaway — APPROVED** [review]

**Review by jeromehardaway — APPROVED** [review]

File: src/pages/testimonials.js (hunk 1 above)
Comment (jeromehardaway) [reply src/pages/testimonials.js]: This is resolved

**Pull request merged**